src/resources/SearchUsageMetrics/RawMetrics/RawMetricsInterface.ts

```diff
@@ -0,0 +1,136 @@
+import {SearchHubRawMetrics} from '../../Enums';
+
+export interface DailyRawMetricParameters extends CommonMetricParameters {
+    /**
+     * The last day to report for
+     * Format YYYY-MM-DD
+     */
+    to: string;
+
+    /**
+     * The first day to report for
+     * Format YYYY-MM-DD
+     */
+    from: string;
+}
+
+export interface MonthlyRawMetricParameters extends CommonMetricParameters {
+    /**
+     * The last day to report for
+     * Format YYYY-MM
+     */
+    to: string;
+
+    /**
+     * The first day to report for
+     * Format YYYY-MM
+     */
+    from: string;
+}
+
+export interface MonthlyRawMetricsParameters {
+    /**
+     * The month to report for
+     * Format YYYY-MM
+     */
+    month: string;
+
+    /**
+     * The minimum number of queries required for a search hub to be listed
+     */
+    minimumQueries?: string;
+}
+
+export interface RestListOfRawMetrics {
+    /**
+     * The list of metrics
+     */
+    metrics: RestMetric[];
+}
+
+export interface RestListOfRawMetricValues {
+    /**
+     * The list of daily metric values
+     */
+    values: RestMetricValueForDate[];
+}
+
+export interface RestListOfSearchHubRawMetrics {
+    /**
+     * The list of search hubs
+     */
+    searchHubs: RestSearchHubMetrics[];
+}
+
+interface RestMetric {
+    /**
+     * The ID of the metric
+     */
+    id: SearchHubRawMetrics;
+
+    /**
+     * The display name of the metric
+     */
+    label?: string;
+}
+
+interface RestMetricValueForDate {
+    /**
+     * The date for the usage metric value
+     */
+    date: string;
+
+    /**
+     * The numeric metric value (int64)
```

Comment:
Not sure if we want to expose the int64 to the public?

Can you check what's done in the rest of the API supported by the platform client, and also is there value for the user to know that?